boa_engine/Cargo.toml

```diff
@@ -56,15 +61,21 @@ unicode-normalization = "0.1.22"
 dyn-clone = "1.0.9"
 once_cell = "1.16.0"
 tap = "1.0.1"
+# intl deps
 sptr = "0.3.2"
 static_assertions = "1.1.0"
 thiserror = "1.0.37"
```

Comment:
Are these only used for intl? If so, shouldn't they be optional?

Reply:
Nope, the comment just shifted upwards when I rebased.